TTS/bin/train_encoder.py

```diff
@@ -212,12 +213,10 @@ def main(args):  # pylint: disable=redefined-outer-name
     try:
         main(args)
     except KeyboardInterrupt:
-        remove_experiment_folder(OUT_PATH)
```

Comment:
The experiment folder is created once the first artifact is written, so this is no longer needed.

Reply:
how and where ? :)

Reply:
originally It does not wait for an artifact but a checkpoint file. 

So it helps to remove the folder if you ctrl+c before there is a checkpoint

Reply:
`fsspec` actually creates the directory structure as needed when writing to paths, so it is created together with the first file. Though I'm not sure, I think that is when the config is written.

But you're right, it means that it no longer removes the folder if aborted after the config is written and before a checkpoint is created. Would you like me to try and re-implement the remove folder function but for fsspec? 

Personally I thought this behaviour really made sense, I this it's a good idea to keep the configs around even if aborted before a checkpoint is saved. E.g. even if I have no checkpoints there might be tensorboard logs with metrics I might want to examine.

Reply:
It makes the developer's life a lot harder and leaves too many marks after running the tests. I'd prefer to keep the behavior as before. In the end, you will have a checkpoint just after the first epoch.

Reply:
Alright, I've restored it now.